### ISSUE TITLE
bad value of aria-current in Scrollspy

### ISSUE THREAD

**Issue by Aniort** [Orange-OpenSource/Orange-Boosted-Bootstrap#455]
hello,
in the Scrollspy component, the `aria-current` attribute for the active element is `page` where it should be `location`.
thanks
